packages/exoflex/src/components/Switch.tsx

```diff
@@ -22,6 +24,21 @@ export type SwitchProps = AccessibilityProps & {
 
 const MARGIN = 2;
 
+export function getTrueWidth(
+  baseWidth: number,
+  style?: StyleProp<ViewStyle>,
+): number {
+  let flattenedStyle = StyleSheet.flatten(style);
+  let width = +(flattenedStyle?.width ?? baseWidth);
+  let minWidth = flattenedStyle?.minWidth;
+  let maxWidth = flattenedStyle?.maxWidth;
+
+  if (!!!minWidth && !!!maxWidth) {
```

Comment:
```suggestion
  if (!minWidth && !maxWidth) {
```

Reply:
I like my condition to be absolute **true/false** rather than *truthy/falsey*.

Reply:
If so, then maybe we should use `typeof` instead of `!!`?

Reply:
`typeof` is too long for that, so it's better to use `boolean cast`.

Reply:
Huh? But the boolean cast here is just a NOT operator
```ts
!!!value === !(!(!value))
// If we look at the individual operation,
// The first one that get executed is
!value
// And that is already utilizing truthiness of the value,
// Turning value into a boolean.
```

I'm not sure if we ever want to do `!!boolean`, it doesn't have any practical effect isn't it? Or maybe there's some hidden trick here?

Reply:
Well, that's true. I guess we don't need to cast it unless the condition is comparing with other value.